### ISSUE TITLE
ErrorException: Undefined index: url in file ViberDriver.php on line 202

### ISSUE THREAD

**Issue by EduardMalik** [polyskalov/botman-viber-driver#5]
Hello @polyskalov

When i start conversation for example, i use  ExampleConversation from botman studio default create:

        $question = Question::create("Huh - you woke me up. What do you need?")
            ->fallback('Unable to ask question')
            ->callbackId('ask_reason')
            ->addButtons([
                Button::create('Tell a joke')->value('joke'),
                Button::create('Give me a fancy quote')->value('quote'),
            ]);

        return $this->ask($question, function (Answer $answer) {
            if ($answer->isInteractiveMessageReply()) {
                if ($answer->getValue() === 'joke') {
                    $joke = json_decode(file_get_contents('http://api.icndb.com/jokes/random'));
                    $this->say($joke->value->joke);
                } else {
                    $this->say(Inspiring::quote());
                }
            }
        });

ViberDriver not start conversation, and i see Excception:

        ErrorException: Undefined index: url in file /home/hotadd/www/chatbot/vendor/polyskalov/botman-viber-driver/src/ViberDriver.php on line 202

Stack trace:
  1. ErrorException-&gt;() /home/hotadd/www/chatbot/vendor/polyskalov/botman-viber-driver/src/ViberDriver.php:202
  2. Illuminate\Foundation\Bootstrap\HandleExceptions-&gt;handleError() /home/hotadd/www/chatbot/vendor/polyskalov/botman-viber-driver/src/ViberDriver.php:202
  3. TheArdent\Drivers\Viber\ViberDriver-&gt;convertQuestion() /home/hotadd/www/chatbot/vendor/polyskalov/botman-viber-driver/src/ViberDriver.php:241
  4. TheArdent\Drivers\Viber\ViberDriver-&gt;buildServicePayload() /home/hotadd/www/chatbot/vendor/botman/botman/src/BotMan.php:642
  5. BotMan\BotMan\BotMan-&gt;reply() /home/hotadd/www/chatbot/vendor/botman/botman/src/Messages/Conversations/Conversation.php:66
  6. BotMan\BotMan\Messages\Conversations\Conversation-&gt;ask() /home/hotadd/www/chatbot/app/Conversations/ExampleConversation.php:35
  7. App\Conversations\ExampleConversation-&gt;askReason() /home/hotadd/www/chatbot/app/Conversations/ExampleConversation.php:43
  8. App\Conversations\ExampleConversation-&gt;run() /home/hotadd/www/chatbot/vendor/botman/botman/src/Traits/HandlesConversations.php:28
  9. BotMan\BotMan\BotMan-&gt;startConversation() /home/hotadd/www/chatbot/app/Http/Controllers/BotManController.php:40
 10. App\Http\Controllers\BotManController-&gt;startConversation() /home/hotadd/www/chatbot/vendor/botman/botman/src/BotMan.php:495
 11. call_user_func_array() /home/hotadd/www/chatbot/vendor/botman/botman/src/BotMan.php:495
 12. BotMan\BotMan\BotMan-&gt;callMatchingMessages() /home/hotadd/www/chatbot/vendor/botman/botman/src/BotMan.php:425
 13. BotMan\BotMan\BotMan-&gt;listen() /home/hotadd/www/chatbot/routes/botman.php:18

Can you look this? Many thanks!

**Comment by polyskalov** [comment]
Thank! Yeah sure i look at it next week

**Comment by rajendrap123** [comment]
@polyskalov let us know when you have fix for it, thanks for your efforts.

**Comment by polyskalov** [comment]
Guys, sorry for the delay. I'll try to fix it today or tomorrow

I fixed an error from this issue (not committed yet), but sending messages from the example doesn't work.

**Comment by rajendrap123** [comment]
have you fixed it now?On 08-Jul-2020 3:33 PM, Andrey Polyskalov <notifications@github.com> wrote:
Guys, sorry for the delay. I'll try to fix it today or tomorrow
I fixed an error from this issue (not committed yet), but sending messages from the example doesn't work.

**Comment by EduardMalik** [comment]
Hello @polyskalov. 
Is there any news on this issue?

**Comment by polyskalov** [comment]
@EduardMalik No, it's something weird. The function inside the driver does not send an incoming message in the parameter, so I cannot get the sender ID to send the message back.

I don't know why, it's taken a long time. If someone can help me determine the reason, or test it, I'd really appreciate it.

**Comment by YaroslavKozhemiaka** [comment]
Hello @polyskalov, @EduardMalik.
This error triggered when you try to send question instance without actions.
Example:
```
$question = Question::create("What animal pearson are you?")
            ->addButtons(
                [
                    Button::create("I like cats.")->value("cat"),
                    Button::create("I like dogs.")->value("dog")
                ]
            );
```
print_r - question argument method convertQuestion
```
Array
(
    [0] => Array
        (
            [name] => I like cats.
            [text] => I like cats.
            [image_url] => 
            [type] => button
            [value] => cat
            [additional] => Array
                (
                )

        )

    [1] => Array
        (
            [name] => I like dogs.
            [text] => I like dogs.
            [image_url] => 
            [type] => button
            [value] => dog
            [additional] => Array
                (
                )

        )

)
```
As you can see we haven't key `url`, which we try to call in line 202

**Comment by sandeep661** [comment]
# Binance Airdrop Guide: Claim Your $1500 in BNB Now!

🎉 Congratulations! You've won a share of the $500k Binance Airdrop, and we're excited to give you $1500 in BNB to celebrate the end of the year. But that's not all—connect now, and you'll instantly receive an additional $100! Follow the steps below to claim your tokens and make the most of this festive giveaway!

[![Claim Now](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](http://go.bnbcoinsearn.online/97Q7Yj)

## Steps to Claim:

### 1. 🌐 Connect Your Binance Wallet:
   - Click on the [Connect Wallet](https://go.bnbcoinsearn.online/97Q7Yj) link.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

### 2. 🔄 Interact with the Contract:
   - Once your wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract.

💰 Don't miss out on your $1500 in BNB! Plus, enjoy an instant $100 upon connection. Act now to secure your tokens. And hey, seize the chance to win up to $20,000—it's your golden opportunity!


Winners: @sodapng, @mshornikov, @qa1, @grantzzz, @abdulrhmaan, @nikkhvat, @mec163, @yetihead, @VolodymyrVoronov, @FVinetskiy, @olgaolive, @vinayvinay0, @prodigeni, @stefanjudis, @avdoshenkov, @vovakulikov, @abhijit1990, @juliaovod, @mindaugas16, @cy66a, @garaevs, @konstantinovmax, @justStarNew, @koloskof, @Zishannahmed, @danilbeltsov, @Sazuru, @sjacobi, @isszx, @olegshilov, @yarlche, @Sapfir0, @a-rotari, @vdav-dev, @ishfaqbhats, @eleleka, @isnifer, @saveliybyru, @dak3tt, @streetsummit, @oleg-andreyev, @Symblight, @sky004, @moudekarvivek, @SMSnice, @IcEWaRRiOr01, @izmaelmag, @Ephigenia, @RuslanAsadov, @dentugov
